drv/lpc55-update-server/src/main.rs

```diff
@@ -493,12 +498,48 @@ impl idl::InOrderUpdateImpl for ServerImpl<'_> {
             UpdateState::Finished | UpdateState::NoUpdate => (),
         }
 
-        self.image = match (component, slot) {
+        let image = match (component, slot) {
```

Comment:
any reason to do the `let` binding here?

Reply:
Nope. Fixed.